src/components/updateUser.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import Col from 'react-bootstrap/Col';
+import { updateUser } from '../actions';
+import Error from './error';
+
+// this can be dumb or smart component - connect works with either
+class UpdateUser extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            validated: false,
+        };
+    }
+
+    handleSubmit = (event) => {
+        const form = event.currentTarget;
+        const username = document.getElementById('username').value;
+        const firstName = document.getElementById('first_name').value;
+        const lastName = document.getElementById('last_name').value;
+        const education = document.getElementById('current_education').value;
+        const job = document.getElementById('current_job').value;
```

Comment:
I don't know if this is the exact right list of attributes for the People table. I think you should be updating FirstName, LastName, PersonalDescription, IndustryID, Email, and Password. I don't think we had username, education, or job in the EERD.